frontend/src/Translations/en/register.json

```diff
@@ -30,9 +30,10 @@
 				"uppercase": "Includes uppercase letter"
 			}
 		},
-		"repeatemail": {
-			"label": "Repeat email",
-			"placeholder": "Repeat your email"
+		"uniqueLogin": {
+			"invalid": "Tag can not contain white chars",
```

Comment:
i know that 'can not' and 'cannot' is the same, but 'cannot' is more common so please use this